src/main/java/org/spongepowered/api/text/dialogue/Dialogue.java

```diff
@@ -66,4 +69,24 @@
      */
     Optional<Question> getCurrentQuestionFor(Speaker speaker);
 
+    /**
+     * Stops the dialogue, removing all {@link Speaker}s.
+     *
+     * @param message The message to send as notification to the speakers
+     * @param suppress Whether or not to suppress the next chat message
+     * @throws IllegalStateException If the dialogue is already concluded
+     */
+    void conclude(Text message, boolean suppress);
```

Comment:
The DataContainer can be gotten from the Dialogue. It hasn't been destroyed or anything.